Lib/unittest/async_case.py

```diff
@@ -0,0 +1,143 @@
+import asyncio
+import inspect
+
+from .case import TestCase
+
+
+
+class AsyncioTestCase(TestCase):
```

Comment:
Can we call it `IsolatedAsyncioTestCase` to signify that the loop is not shared between the tests?

Reply:
We can.
I have a hope that non-isolated tests can reuse the same test case but require a little different test runner maybe (to keep a loop during the whole test run session).

Anyway, I'm ok with renaming the class right now and see what's happen when we add a shared loop environment.

Reply:
Renamed.
Not sure if I like it too much (the long name becomes even longer) but can live with it, sure.

Reply:
I don't like it too much either,  but the predictability of the behaviour of this new base class comes first.